academy/ibc/ibc-token-lifecycle.md

```diff
@@ -0,0 +1,61 @@
+
+The *experience* of receiving and sending updates to these ledgers is so similar to that of working with cashlike instruments, and the intricacies of blockchain mechanics are sufficiently complex that these simplified terms are sufficient for most users. A user-friendly, cognitively light experience is essential to adoption. 
+
+So, while inter-chain transfers of tokens may be described as a "token" crossing a "bridge" to arrive on another chain, deep down we know that what *must* be unfolding is a coordination of updates between two chains. 
+
+## Design challenge
+
+Blockchains are internally, mathematically consistent. Recall that a blockchain is an ordered set of transactions, each signed by the originator. The state of blockchain is determined by applying that set of transactions to a strict set of protocol rules. A blockchain cannot include an external reference to an external data source such as an API. If it did, then it would not be possible to *guarantee* that all properly functions nodes will always be able to eventually agree on the state.
+
+The first challenge is that of binding two ledgers on two blockchains that and do not and cannot communicate with each other. 
+
+Many applications *need* external signals. "Oracles" are a generalized solution to this problem. Oracles take many forms. In general, an "Oracle" is a regular account that is trusted to observe the external source, on its own, and then *sign a transaction* that injects the observation into the blockchain system. Blockchain nodes will never be able to verify that the Oracle provided accurate information, but they will be able to verify that Oracle provided a specific piece of information and signed it. 
+
+Therefore, in systems where Oracles are relied upon for a critical input, the trustworthiness of the overall system is no greater than trust in the Oracle. Consequently, such systems tend to focus on designs that encourage strong trust in the Oracle. Generally speaking, bridges are, in part, Oracles that can observe both chains as well in sign transactions and inject information about observations into the other chain. 
+
+Another challenge arises from differences in the way two chains operate. They may have differring block times. Even more importantly, one chain may be *probablistic* while the other is *deterministic*. Consider the following. If chain A releases tokens based on something that happened on chain B and then chain B reverted the original transaction (because chain B is probalistic and this possibility can never be completely eliminated), then a double-spend vulnerability emerges.
+
+## Design constraints
+
+There are obvious usabililty requirements for a successful bridge:
```

Comment:
I would maybe not use the term "bridge". It might be confusing with the Gravity bridge.